docs/api/ShallowWrapper/contains.md

```diff
@@ -56,10 +56,22 @@ expect(wrapper.contains([
 ])).to.equal(false);
 ```
 
+```jsx
+let calculatedValue = 2 + 2;
+
+const wrapper = shallow((
+  <div>
+    <div data-foo="foo" data-bar="bar">{calculatedValue}</div>
+  </div>
+));
+
+expect(wrapper.contains(<div data-foo="foo" data-bar="bar">{4}</div>)).to.equal(true);
+```
 
 #### Common Gotchas
 
 - `.contains()` expects a ReactElement, not a selector (like many other methods). Make sure that
 when you are calling it you are calling it with a ReactElement or a JSX expression.
 - Keep in mind that this method determines equality based on the equality of the node's children as
 well.
+- Embedded expressions must be wrapped with curly braces
```

Comment:
This part, I'm not so sure about - this is just how jsx works, and isn't required if you're using `React.createElement`.

Reply:
I certainly would've picked up my error much earlier with just the example added above, so I'll remove this line.

I think it was just the fact all the sample test JSX happened to be showing valid HTML which reinforced my newbie assumptions :)